exporter/clickhouseexporter/config.go

```diff
@@ -58,13 +58,15 @@ type QueueSettings struct {
 	QueueSize int `mapstructure:"queue_size"`
 }
 
+const defaultDatabase = "default"
+
 var (
 	errConfigNoEndpoint      = errors.New("endpoint must be specified")
 	errConfigInvalidEndpoint = errors.New("endpoint must be url format")
 )
 
-// Validate validates the clickhouse server configuration.
-func (cfg *Config) Validate() (err error) {
+// validate the clickhouse server configuration.
+func (cfg *Config) validate() (err error) {
```

Comment:
If you keep it exported, you don't need to call it explicitly.

Reply:
you are right, I recover it.